packages/task/src/browser/quick-open-task.ts

```diff
@@ -33,15 +32,12 @@ export class QuickOpenTask implements QuickOpenModel, QuickOpenHandler {
     @inject(TaskService)
     protected readonly taskService: TaskService;
 
-    @inject(TaskConfigurations)
-    protected readonly taskConfigurations: TaskConfigurations;
```

Comment:
that's 0.5.0

i'm fine with that, please be conscious when you break

Reply:
you are right. i will put it back. Thank you !

Reply:
There are 2 more breaking changes in this PR.

I'm fine with 0.5.0 for next. There are PRs, like refactoring git to scm extension, for which it is not avoidable.

Reply:
@akosyakov just thinking about API breaking going under the radar, would there be a strategy to adopt in order to catch any API breakage?

I guess this could be done with our test suite, but despite all the tests we have, it seems to not be enough. Is there something that should be done?

Reply:
I'm not aware of any tools for TS allowing to check it. Even if there are I am not sure that we want to use them, since we are fine with breaking and updating the major version. It would cause a lot of noise.

Reply:
I put `taskConfigurations` back and commented `To be removed in 0.5.0`. We could leave it here for the release where other breaking changes are made.